blivet/size.py

```diff
@@ -59,14 +59,21 @@
                    _Prefix(1024**7, N_(b"zebi"), N_(b"Zi")),
                    _Prefix(1024**8, N_(b"yobi"), N_(b"Yi"))]
 
-_bytes = [N_(b'B'), N_(b'b'), N_(b'byte'), N_(b'bytes')]
+# Handle 'B' separately so that it can be localized without translating
+# both 'B' and 'b'
+_bytes_letter = N_(b'B')
+_bytes_words = [N_(b'byte'), N_(b'bytes')]
 _prefixes = _binaryPrefixes + _decimalPrefixes
 
+def _bytes():
+    return [_bytes_letter + _lowerASCII(_bytes_letter)] + _bytes_words
```

Comment:
It doesn't, I suppose. I just did it that way for consistency with _xlated_bytes()